server/repositories/entities/report.ts

```diff
@@ -14,5 +14,5 @@ export default class Report {
   reportDefinition: ReportDefinition
 
   @OneToMany('FieldValue', 'report', { eager: true })
-  fieldValues: Array<FieldValue> | null
+  fieldValues: Array<FieldValue> | []
```

Comment:
Do we even need the Pipe at this point? An empty Array should still be an array? I genuinely don't know here, I had to google it to find out that the pipe means this is a union type - not come across them before! Quite a cool feature!

Reply:
Yep, we don't really need the Pipe